### ISSUE TITLE
Carousel inside an ng-repeated materialize card loses jquery and interaction functionality

### ISSUE THREAD

**Issue by BodibaDev** [Dogfalo/materialize#3896]
Hi, I'm having issues with the carousel whenever it's inside a carousel, it loses functionality..
Initially thought that it was just a clash with AngularJs but I've loaded the libraries properly..

Here's my page
`<!DOCTYPE html>
  <html ng-app="Gatland">
    <head>  
      
      <link href="http://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
      
      <link type="text/css" rel="stylesheet" href="css/materialize.min.css"  media="screen,projection"/>
      <link type="text/css" rel="stylesheet" href="css/photographerpick.css"/>
      <link rel="stylesheet" href="css/lightbox.css">

      <title>Gatland Photography | Style Pick</title>

      <link href="https://fonts.googleapis.com/css?family=Quicksand:300,400" rel="stylesheet">
      
      <meta name="viewport" content="width=device-width, initial-scale=1.0"/>

    </head>

    <body ng-controller="photographerController">
      
      <script type="text/javascript" src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
      <script type="text/javascript" src="js/materialize.min.js"></script>
      

      
      <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular.min.js"></script>
      <script src="controllers/app.js"></script>
      <script src="controllers/photographerController.js"></script>
      <script src="services/jobService.js"></script>

       
    
      <div class="navbar-fixed"> 
       <nav class="z-depth-0">
        <div class="nav-wrapper"  style="background-color:#F5F5EE;">
          <div class="container">
              <a href="#" class="brand-logo center" style="color: #1A4F61;"> <span class="headerResponsive">{{piki}}</span>
              
              
              </a>        
            <a href="province.html">
              <i class="material-icons" style="color: #1A4F61;">replay</i>
            </a>
          </div>
        </div>
       </nav>
     </div>
       
  
       
      <div class="container">

      <div class="card-panel" ng-repeat="i in photographersList track by i._id" style="padding-left: 0px;padding-top: 0px;padding-right: 0px;padding-bottom: 0px;">
        <div class="card-content" >
          <div class="row" style="margin-bottom: 0px;padding-left: 20px;padding-top: 20px;padding-right: 20px;padding-bottom: 10px;">

          
            <div class="col s1">
              <img ng-src="{{i.profilepic}}" alt="" class="circle responsive-img" style="height: auto;width: 100%;">
            </div> 

          

          
            <div class="col s11">
              <div class="carousel" style="height: 138px;">
                <a class="carousel-item" href="#one!"><img ng-src="{{i.weddingPhotos[0]}}"></a>
                <a class="carousel-item" href="#two!"><img ng-src="{{i.weddingPhotos[1]}}"></a>
                <a class="carousel-item" href="#three!"><img ng-src="{{i.weddingPhotos[2]}}"></a>
                <a class="carousel-item" href="#four!"><img ng-src="{{i.weddingPhotos[3]}}"></a>
                <a class="carousel-item" href="#five!"><img ng-src="{{i.weddingPhotos[4]}}"></a>
                <a class="carousel-item" href="#six!"><img ng-src="{{i.weddingPhotos[5]}}"></a>
                <a class="carousel-item" href="#seven!"><img ng-src="{{i.weddingPhotos[6]}}"></a>
                <a class="carousel-item" href="#eight!"><img ng-src="{{i.weddingPhotos[7]}}"></a>
                <a class="carousel-item" href="#nine!"><img ng-src="{{i.weddingPhotos[8]}}"></a>
                <a class="carousel-item" href="#nine!"><img ng-src="{{i.weddingPhotos[9]}}"></a>
              </div>
               <br>
            </div>
           
          

            
              <div class="card-action noFilter" style="padding-top: 0px;padding-right: 0px;padding-bottom: 0px;padding-left: 0px;border-top-width: 0px;">
              <a ng-class="{grey: GreyisTrue}" ng-click="clickedGuy();addToPicks($index)" class="waves-effect waves-light btn-large noFilter" style="height: auto;width: 100%;background-color: #80A6B3;font-family:Quicksand;font-size: 1.4em;">{{pickingGuider}}</a>
            </div> 
            
        </div>

      </div>
      </div>  

      <br><br><br>
    
        <footer style="position: fixed;left:0px;bottom:0px;width:100%; z-index: 1;">
          <a ng-href="{{finalformGateway}}" ng-click="submitClearService()" ng-disable="true" class="btn waves-effect waves-light" type="submit" name="action" style="height: auto;width: 100%;background-color: #80A6B3;font-size: 2em;padding-bottom: 10px;padding-top: 10px;">Submit</a>
        </footer>

    </body>
  </html>
`

Here's my carousel function 
`
$( document ).ready(function(){
	$(".button").sideNav();
	$('.datepicker').pickadate({
    selectMonths: true, // Creates a dropdown to control month
    selectYears: 15 // Creates a dropdown of 15 years to control year
  	});
  	$('select').material_select();

  	console.log('About to carousel')
  	 $('.carousel').carousel('next');
  	 $('.carousel').carousel({
  	 	full_width: true,
  	 	padding: 20
  	 });
  	
  	console.log('I carouseled')
})

`

Here's a link to the app (gatland.streetcapital.co). Pick 'Wedding' then 'Eastern Cape' to recreate the problem..

**Comment by fega** [comment]
Closed due inactivity, feel free to reopen it if it is still necessary.
